.gitignore

```diff
@@ -35,3 +35,4 @@ yarn-error.log*
 
 report.*.json
 /rmwc
+generated-*
```

Comment:
Removing any generated files from the repo and rebuilding them as needed. This will avoid conflicts in the future and keep the codebase clean.